CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ Here is a template for new release sections
 - Test input files (#343)
 - All parameters of the json/csv input files are now defined by constant variables (i.e, `CRATE="crate"` instead of string `"crate"`) (#346)
 - Use "is" instead of "==" in if clauses for True, False and None (#346)
+- Categorize constants in 'constants_json_strings.py' (#347)
+- Renaming CAPEX_FIX = "capex_fix" into C_DEVELOPMENT = "Cost_development" (#40)
+- Renaming CAPEX_VAR = "capex_var" into C_SPECIFIC = "cost_specific" (#40)
```

Comment:
The PR number is #347 not #40 which is the issue number. My rationale here is that you want to indicate where you did the change in the CHANGELOG, not why (why is accessible via the PR)

Reply:
Ah, ups, then I did this wrong all the time. Thanks!